includes/Stories_Renderer/Renderer.php

```diff
@@ -129,10 +144,18 @@ abstract class Renderer implements RenderingInterface, Iterator {
 	 * @param Stories $stories Stories instance.
 	 */
 	public function __construct( Stories $stories ) {
-
 		$this->stories         = $stories;
 		$this->attributes      = $this->stories->get_story_attributes();
-		$this->content_overlay = $this->attributes['show_title'] || $this->attributes['show_date'] || $this->attributes['show_author'];
+		$this->content_overlay = $this->attributes['show_title'] || $this->attributes['show_date'] || $this->attributes['show_author'] || $this->attributes['show_excerpt'];
+	}
+
+	/**
+	 * Get current object id.
+	 *
+	 * @return int Object id.
+	 */
+	protected static function get_instance_id() {
```

Comment:
Is this method used anywhere? Can't find references.

Reply:
not anymore, will have to remove. thanks.